README.md

```diff
@@ -1,79 +1,72 @@
 
-* Enter your github username
-* Enter the repository you want to train the model on eg: `community`
-* Enter the organization the repo belongs to eg: `operate-first`
-* Enter the personal access token generated in the previous step eg: `ghp_xyzxyzxyz`
+This mode will initiate the model training process by following the steps of data collection, feature engineering, model training on the PR data available and finally running the inference i.e. predicting the time to merge for the latest PR on the repository. 
 
-If you are passing your S3 credentials here
-* Enter your bucket name
-* Enter your endpoint url
-* Enter your Access Key
-* Enter your Secret Key
+(*NOTE : This workflow will fail if there are no PRs on the repository*)
 
-![image](https://user-images.githubusercontent.com/26301643/196466088-10e8f725-0e5c-494e-b146-a1fd5ce6c31e.png)
+You can also initiate a new trigger by going to actions for your repository like [here](https://github.com/aicoe-aiops/ocp-ci-analysis/actions/workflows/train-ttm.yaml):
 
-2. **Manual Trigger**
+![image](https://user-images.githubusercontent.com/26301643/206544812-b6ffbe44-7bd3-4c7d-ab75-55b29d24f8f4.png)
 
-To trigger manually, first set all the environment variables needed to run the workflow as Action secrets to the repo.
-These include `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY`, `S3_BUCKET`, `S3_ENDPOINT_URL`, `CEPH_BUCKET_PREFIX`, `REPO` and `ORG`.
+Go select - `Run Time to Merge Model Training` and go to `Run workflow` on upper right and run it like such :
 
-Now, go to the "Actions" section of your forked repository and click on workflow "Run in container" and click on "Run workflow"
+![image](https://user-images.githubusercontent.com/26301643/206548995-5e82cf06-1892-462d-9114-e81065b1f13e.png))
 
-<img width="500" alt="image" src="https://user-images.githubusercontent.com/32435206/195928717-079b0c85-c953-43a9-b6e4-cf6efbc7dff5.png">
+This will initiate the model training and inference action. 
 
-## Step 4
 
-To view your running workflow from the Github UI, go to "Actions" and click on the workflow run
+2. **Infernce Mode** : 
 
-<img width="700" alt="image" src="https://user-images.githubusercontent.com/32435206/196230588-d8485c8c-2f97-46a9-ba07-b86e1b7c6782.png">
+Similar to the `train-ttm.yaml` file, you can add another file called `predict-ttm.yaml` file to `.github/worklows/` like [this](https://github.com/aicoe-aiops/ocp-ci-analysis/blob/master/.github/workflows/predict-ttm.yaml). This file has `MODE` as `0` which would enable just inference on the new incoming Pull Request and add a comment to the pull request specifying the approximate time it will take to be merged.
 
```

Comment:
Do you want to add a section here to say if you would like to try this tool out without adding the workflow file to your repository. And keep a short version of some of the instructions from the previous version. This might be useful fork forks who just want to try this out without setting this up in their repo. If that is making the readme too big, maybe you can pull that out into a different doc and point to the detailed instructions in the previous version of the readme from this section keeping this here short.

Alternative method

You can fork this repository and add secrets your fork and run the training by using the bash script or by manually triggering the workflow....